### DISCUSSION TITLE
Old "Primefaces - Forum"

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [primefaces/community#21] [PrimeFaces] [answered]
Are all of the old "Primefaces - Forum" postings being migrated to GitHub:

https://github.com/orgs/primefaces/discussions/categories/primefaces

**Comment by atkntepe** [comment, marked as answer]
Hi, NOTiFY. We made the old forum read-only. We dont have a plan to migrate the old forum discussions here. As you can imagine, there were thousands of them. I know you have hundreds of answers there, but this will be the new place for our forum. I am sure this place will be packed within weeks. 

We needed to change it for various reasons, and in the end, we decided on this place and the structure to monitor and provide more efficiently.

Welcome to our new place.
